src/Database/Adapter.php

```diff
@@ -66,7 +66,7 @@ public function resetDebug(): self
     public function setNamespace(string $namespace): bool
     {
         if (empty($namespace)) {
-            throw new Exception('Missing namespace');
+            throw new DatabaseException('Missing namespace');
```

Comment:
Lets revert the changes in this file. Missing namespace, missing database etc are not under the control of the end-user